docs/Learning-Environment-Design.md

```diff
@@ -114,17 +112,21 @@ the Academy properties and their uses.
 
 ### Brain
 
-The Brain encapsulates the decision making process. Brain objects must be
-children of the Academy in the Unity scene hierarchy. Every Agent must be
+The Brain encapsulates the decision making process. Every Agent must be
 assigned a Brain, but you can use the same Brain with more than one Agent.
-
-Use the Brain class directly, rather than a subclass. Brain behavior is
-determined by the Brain type. During training, set your Agent's Brain type to
-**External**. To use the trained model, import the model file into the Unity
-project and change the Brain type to **Internal**. See
+__Note__:You can assign the same brain to multiple agents by using prefabs
+or by selecting all the agents you want to attach the brain to using the 
+search bar on top of the Scene Hierarchy window.
+
+To Create a Brain, go to `Assets -> Create -> Ml-Agents` and select the 
+type of brain you want to use. During training, use a **Learning Brain** 
+and drag it into the Academy's `Broadcast Hub` with the `Control` checkbox checked.
+When you want to use the trained model, import the model file into the Unity
+project, add it to the **Model** property of the **Learning Brain** and uncheck
+the `Control` checkbox of the `Broadcast Hub`. See
 [Brains](Learning-Environment-Design-Brains.md) for details on using the
-different types of Brains. You can extend the CoreBrain class to create
-different Brain types if the four built-in types don't do what you need.
+different types of Brains. You can create new kinds of brains if the three
```

Comment:
What other brains can we build?

Reply:
I do not know, you could imagine a brain that would record all the actions perform and store them to disk for example.